test/libsolidity/StandardCompiler.cpp

```diff
@@ -181,6 +181,12 @@ BOOST_AUTO_TEST_CASE(basic_compilation)
 			"fileA": {
 				"content": "contract A { }"
 			}
+		},
+		"outputSelection": {
+			"fileA": {
+				"A": [ "abi", "devdoc", "userdoc", "evm", "metadata" ]
```

Comment:
This is annoying, one of the reasons for the failing tests was a missing comma here at the end 😕